docker-compose.yml

```
@@ -8,14 +8,15 @@ services:
   #   ports:
   #     - ${APP_PORT}:${APP_PORT}
   db:
-    image: mysql:5.7.30
+    image: mariadb:latest
     command: mysqld --sql_mode="" --init-file /data/application/init.sql
     restart: always
     volumes:
       - ./volumes/mysql:/var/lib/mysql
       - ./settings/mysql/init.sql:/data/application/init.sql
     environment:
       MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
+      MYSQL_DATABASE: ${DATABASE}
```

```suggestion
      MYSQL_DATABASE: ${MYSQL_DATABASE}
```
Also change .env 3QQ